codex-rs/core/tests/suite/user_shell_cmd.rs

```diff
@@ -0,0 +1,119 @@
+#![cfg(unix)]
```

Comment:
Given these tests are limited to unix for now (which I would prefer not to do), do we really need `CODEX_HERMETIC_TEST_SHELL`?